### PR TITLE
doc: small fix: add localtoc to RST

### PR DESCRIPTION
**Pull request by FrancescoSer**
Added Local ToC tree to all RST pages.
JIRA NCSDK 6534.

Signed-off-by: Francesco Servidio <francesco.servidio@nordicsemi.no>

### CODE REVIEW

File: applications/serial_lte_modem/doc/slm_description.rst [hunk 1]
```diff
@@ -3,8 +3,11 @@
 Application description
 #######################
 
-The Serial LTE Modem (SLM) application demonstrates how to use the nRF9160 as a stand-alone LTE modem that can be controlled by proprietary AT commands.
+.. contents::
+   :local:
+   :depth: 2
```
Comment (ru-fu) [inline applications/serial_lte_modem/doc/slm_description.rst:8]:
```suggestion
   :depth: 3
```
That would make it more usable - but I don't know if we want to be consistent with the level here.

**Review by MarekPorwisz — APPROVED** [review]

**Review by trond-snekvik — APPROVED** [review]

**Review by joerchan — APPROVED** [review]

**Review by tejlmand — APPROVED** [review]

File: applications/serial_lte_modem/doc/slm_description.rst (hunk 1 above)
Comment (FrancescoSer) [reply applications/serial_lte_modem/doc/slm_description.rst:8]: As just discussed, we will fix this in a different PR.

File: doc/static/css/common.css [hunk 2]
```diff
@@ -226,7 +226,7 @@ div.numbered-step>h6::before {
     border: 1px solid #ddd;
     background-color: #fafafa;
     padding: 10px 10px 0px 0px;
-    margin: 5px;
+    margin: 25px;
```
Comment (ru-fu) [inline doc/static/css/common.css:229]: Sorry, I was lazy - but 25px on all sides looks bad, we only want it on the left and on the bottom.
So:
```suggestion
    margin: 0px 0px 25px 25px;
```

Comment (FrancescoSer) [reply doc/static/css/common.css]: fixed

**Review by pdunaj — APPROVED** [review]

**Review by ru-fu — APPROVED** [review]
Need to update the manifest again after https://github.com/nrfconnect/sdk-nrfxlib/pull/301 is merged.

**Comment by tejlmand** [comment]
@FrancescoSer Why was `DNM` removed when manifest still contains `pull/301/head` ?
image

**Comment by tejlmand** [comment]
Please update SHA: c8530e1d39e868e4077a3a4b8a5f6e1f6cba9289

**Review by tejlmand — APPROVED** [review]

**Pull request merged**